code/game/machinery/Sleeper.dm

```diff
@@ -33,6 +33,11 @@
 
 /obj/machinery/sleeper/examine(mob/user)
 	. = ..()
+	if(occupant)
+		if(occupant.is_dead())
+			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE, FALSE)] is dead!</span>"
+		else
+			. += "You see [occupant.name] inside."
```

Comment:
Could you use `<span class='notice'>` here to differentiate this from the rest of the examine text?